src/probability/variance.lean

```diff
@@ -30,28 +39,166 @@ open_locale big_operators measure_theory probability_theory ennreal nnreal
 
 namespace probability_theory
 
-/-- The variance of a random variable is `𝔼[X^2] - 𝔼[X]^2` or, equivalently, `𝔼[(X - 𝔼[X])^2]`. We
-use the latter as the definition, to ensure better behavior even in garbage situations. -/
+/-- The `ℝ≥0∞`-valued variance of a real-valued random variable defined as a lintegral.-/
```

Comment:
```suggestion
/-- The `ℝ≥0∞`-valued variance of a real-valued random variable defined as the Lebesgue integral of `(X - 𝔼[X])^2`. -/
```
Also that file uses both `f` and `X` for the random variable. Could you change all of them to `X`?

Reply:
I used the same notations as the original variance file: `f` when the measure is explicit and `X` when the measure is volume. I quite like this distinction. Do you think it's better to use only `X`?

Reply:
It looked weird to me because apparently the file used `X` in docstrings for lemmas that used `f` in the code. I'd rather have `X` everywhere. I don't get why there should be a distinction between volume or another measure.

Reply:
Alright, I've changed it all to `X` now